armi/utils/tests/test_densityTools.py

```diff
@@ -108,6 +122,13 @@ def test_getMassFractions(self):
         self.assertAlmostEqual(massFracs["U234"], 0.07937081219437897)
 
     def test_calculateNumberDensity(self):
+        """
+        Mass fraction to number density.
+
+        .. test:: Number densities from mass fraction
+            :id: T_ARMI_UTIL_MASS2N_DENS1
+            :tests: R_ARMI_UTIL_MASS2N_DENS
```

Comment:
We should remove this `..test::` tag.

This req is only implemented by the function `getNDensFromMasses()`, so for this req we only need to worry about tests for that function.